cli/commands/migrate_apply.go

```diff
@@ -34,6 +34,7 @@ func newMigrateApplyCmd(ec *cli.ExecutionContext) *cobra.Command {
 	f.StringVar(&opts.downMigration, "down", "", "apply all or N down migration steps")
 	f.StringVar(&opts.versionMigration, "version", "", "migrate the database to a specific version")
 	f.StringVar(&opts.migrationType, "type", "up", "type of migration (up, down) to be used with version flag")
+	f.BoolVar(&opts.dryRun, "dry-run", false, "")
```

Comment:
Let's call this `skip-execution`: `skip executing the migration action, but mark them as applied`